viewing_party/party.py

```diff
@@ -1,23 +1,183 @@
 # ------------- WAVE 1 --------------------
 
-def create_movie(title, genre, rating):
-    pass
+def create_movie(movie_title, genre, rating):
+    if movie_title == None or genre == None or rating == None:
+        return None
+    movie = {"title":movie_title, "genre":genre, "rating":rating}
+    return movie
 
+def add_to_watched(user_data, movie):
+    if movie["title"] not in user_data["watched"]:
+        user_data["watched"].append(movie)
+    return user_data
+    
+def add_to_watchlist(user_data, movie):
+    if movie["title"] not in user_data["watchlist"]:
+        user_data["watchlist"].append(movie)
+    return user_data
+    
+def watch_movie(user_data, movie_title):
+    for movie in user_data["watchlist"]:
+        if movie_title == movie["title"] and movie_title not in user_data["watched"]:

```

Comment:
👀 There is no requirement in the documentation that says that we should check that a movie is not in the "watched" list so the second check on line 21 needs to be removed. 

Again, if there's ambiguity about project please reach out for clarification. It's not a huge deal in the grand scheme of things here because this is a practice project at Ada. 

However, making logical changes like this when they weren't part of the project requirements while you're in internship or in your full time role could mean actual differences in how an app behaves (which ultimately could lead to a poor customer experience of losing money). It will be critical as a dev to confirm that your code aligns with what your product manager has outlined.

```suggestion
        if movie_title == movie["title"]:
```